app/routes/poll-network.tsx

```diff
@@ -16,9 +16,14 @@ export async function loader(): Promise<LoaderData> {
   }
 }
 
+const LS_KEY = "flow_network_data"
+
 export default function () {
   const networkData = useLoaderData<LoaderData>()
-  const [status, setStatus] = useState(networkData)
+  const cachedStatus = localStorage.getItem(LS_KEY)
```

Comment:
This won't work if rendered server-side, will it? I don't think `localStorage` will be available on a server render. https://remix.run/docs/en/v1/guides/constraints#rendering-with-browser-only-apis

Reply:
I'll just take this out for now. Keep forgetting remix does SSR.